From: Alexander Graf <agraf@suse.de>
Date: Wed, 11 Jun 2014 10:39:38 +0200
Subject: KVM: PPC: Book3S HV: Enable for little endian hosts
Origin: https://git.kernel.org/linus/6947f948f06128409b94306afaca5ece873ee5a2

Now that we've fixed all the issues that HV KVM code had on little endian
hosts, we can enable it in the kernel configuration for users to play with.

Signed-off-by: Alexander Graf <agraf@suse.de>
---
 arch/powerpc/kvm/Kconfig |    1 -
 1 file changed, 1 deletion(-)

Index: tree-jessie/arch/powerpc/kvm/Kconfig
===================================================================
--- tree-jessie.orig/arch/powerpc/kvm/Kconfig
+++ tree-jessie/arch/powerpc/kvm/Kconfig
@@ -75,7 +75,6 @@ config KVM_BOOK3S_64
 config KVM_BOOK3S_64_HV
 	tristate "KVM support for POWER7 and PPC970 using hypervisor mode in host"
 	depends on KVM_BOOK3S_64
-	depends on !CPU_LITTLE_ENDIAN
 	select KVM_BOOK3S_HV_POSSIBLE
 	select MMU_NOTIFIER
 	select CMA
